python/phylanx/ast/physl.py

```diff
@@ -632,7 +632,10 @@ def _Return(self, node):
             end of the function!
         """
 
-        return self.apply_rule(node.value)
+        if type(node.value) == ast.Tuple:
+            return ["make_list", self.apply_rule(node.value)]
```

Comment:
I think we're trying to move away from `make_list()` in favor of just `list()`. Would you mind changing this?